consensus/merge/src/main/java/org/hyperledger/besu/consensus/merge/blockcreation/MergeCoordinator.java

```diff
@@ -82,7 +82,7 @@ public class MergeCoordinator implements MergeMiningCoordinator, BadChainListene
 
   private static final long DEFAULT_TARGET_GAS_LIMIT = 36_000_000L;
   // testnets might have higher gas limits than mainnet
-  private static final long DEFAULT_TARGET_GAS_LIMIT_TESTNET = 36_000_000L;
+  private static final long DEFAULT_TARGET_GAS_LIMIT_TESTNET = 60_000_000L;
```

Comment:
The current plan was to only increase Sepolia to 60M, but leave Hoodi at the current level. So I think it's better to have a separate constant for every public testnet.